package.json

```diff
@@ -36,39 +36,43 @@
     "outputPath": "./bin"
   },
   "devDependencies": {
+    "@jest/types": "^27.4.2",
     "@types/jest": "^27.0.3",
     "@types/jest-dev-server": "^5.0.0",
     "@types/node": "^16.11.13",
+    "@types/uuid": "^8.3.3",
     "@typescript-eslint/eslint-plugin": "^4.29.3",
     "@typescript-eslint/parser": "^4.29.3",
     "conventional-changelog": "^3.1.24",
     "cross-env": "^7.0.3",
     "eslint": "^7.32.0",
     "eslint-config-prettier": "^8.3.0",
+    "eslint-import-resolver-node": "^0.3.6",
     "eslint-plugin-import": "^2.24.1",
     "eslint-plugin-jsdoc": "35.5.1",
     "fs-extra": "^10.0.0",
     "jest": "^27.4.5",
     "jest-dev-server": "^6.0.3",
-    "parcel": "1.12.3",
+    "jest-mock": "^27.4.2",
     "pkg": "^5.5.1",
+    "portfinder": "^1.0.28",
```

Comment:
Why not?

Reply:
Because only the server package uses it and already has it as its dependency (https://github.com/botpress/messaging/blob/sm-yarn-3/packages/server/package.json#L42)